dietpi/dietpi-sync

```diff
@@ -140,79 +140,71 @@ _EOF_
 			Create_Filter_Include_Exclude
 
 			#Rsync options
-			local rync_options+="-aHP4 --info=name0 --info=progress2 --human-readable --delete-excluded --exclude-from=$FP_FILTER_INCLUDE_EXCLUDE --log-file=$FP_LOG "
+			local rync_options="-aHP4h --info=name0 --info=progress2 --delete-excluded --exclude-from=$FP_FILTER_INCLUDE_EXCLUDE --log-file=$FP_LOG"
```

Comment:
@Fourdee 
Due to `--delete-excluded`, the sync status file is deleted as well, so only the last status log will be present. Not a big issue for me, but due to grep/tail combi to check last status, I guess this was not intended?

The /var/log/dietpi-sync.log on the other hand will grow, if not dietpi-ramlog in use. Since this contains many lines on large sync dirs, perhaps we should clear it at sync start?

Reply:
@MichaIng 

> Or actually for my impression we could merge both into a single log file. To preserve status (with ramlog) e.g. log everything to .dietpi-sync.log (tmp working dir), incl. final status output, then mv .dietpi-sync.log $FP_TARGET/?

Yep good idea 👍.